dll/cpl/sysdm/lang/ro-RO.rc

```diff
@@ -312,13 +312,13 @@ BEGIN
     SCROLLBAR IDC_DIALOG_GRIP, 293, 243, 7, 7, SBS_SIZEGRIP
     CONTROL "", IDC_LIST_VARIABLE_VALUE, "SysListView32", LVS_NOCOLUMNHEADER | LVS_EDITLABELS | LVS_SHOWSELALWAYS |
                 LVS_SINGLESEL | LVS_REPORT | WS_BORDER | WS_TABSTOP, 10, 10, 224, 208, WS_EX_CLIENTEDGE
-    PUSHBUTTON "&New", IDC_BUTTON_NEW, 242, 10, 50, 14
-    PUSHBUTTON "&Edit", IDC_BUTTON_EDIT, 242, 30, 50, 14
-    PUSHBUTTON "&Browse...", IDC_BUTTON_BROWSE_FOLDER, 242, 50, 50, 14
-    PUSHBUTTON "&Delete", IDC_BUTTON_DELETE, 242, 70, 50, 14
-    PUSHBUTTON "Move &Up", IDC_BUTTON_MOVE_UP, 242, 100, 50, 14
-    PUSHBUTTON "Move D&own", IDC_BUTTON_MOVE_DOWN, 242, 120, 50, 14
-    PUSHBUTTON "Edit &text...", IDC_BUTTON_EDIT_TEXT, 242, 150, 50, 14
+    PUSHBUTTON "No&u", IDC_BUTTON_NEW, 242, 10, 50, 14
+    PUSHBUTTON "&Editare", IDC_BUTTON_EDIT, 242, 30, 50, 14
+    PUSHBUTTON "Na&vigare...", IDC_BUTTON_BROWSE_FOLDER, 242, 50, 50, 14
+    PUSHBUTTON "Șt&ergere", IDC_BUTTON_DELETE, 242, 70, 50, 14
```

Comment:
The access key for „e” is still assigned twice (down from three).

Reply:
It's fixed now.